cmd/buildtools/utils.go

```diff
@@ -160,3 +165,77 @@ func MirrorChart(repo, name, ver string) error {
 	logrus.Infof("pushed openebs chart: %s", remote)
 	return nil
 }
+
+func DownloadFile(url, dest string) error {
+	resp, err := http.Get(url)
+	if err != nil {
+		return fmt.Errorf("unable to get %s: %w", url, err)
+	}
+	defer resp.Body.Close()
+
+	out, err := os.Create(dest)
+	if err != nil {
+		return fmt.Errorf("unable to create %s: %w", dest, err)
+	}
+	defer out.Close()
+
+	_, err = io.Copy(out, resp.Body)
+	return err
+}
+
+func ExtractTGZArchive(tgzFile string, destDir string) error {
+	fileReader, err := os.Open(tgzFile)
+	if err != nil {
+		return fmt.Errorf("failed to open tgz file %q: %w", tgzFile, err)
```

Comment:
why mix "unable to" and "failed to" in the same file? maybe change these all to "unable to". maybe for a discussion in the future but i prefer it without the prefix so reading a wrapped error message doesnt repeat it over and over again

```suggestion
		return fmt.Errorf("open tgz file %q: %w", tgzFile, err)
```